roles/openshift_clock_facts/tasks/main.yaml

```diff
@@ -0,0 +1,40 @@
+---
+- fail:
+    msg: "openshift_clock_method must be set to either 'none', 'auto', 'ntpd', or 'chrony'"
+  when:  openshift_clock_method is defined and
+    openshift_clock_method not in ["none", "auto", "ntpd", "chrony"]
+
+- set_fact:
+    openshift_clock_method: chrony
+  when: openshift_clock_method is defined and openshift_clock_method == "auto" and
+    openshift.common.is_containerized and openshift.common.is_atomic
```

Comment:
hmm, I'm wondering if this test is going to be sufficient. We know that when `is_atomic == True` that chrony is the only available option, but `is_containerized` could also mean a RHEL7 host with ntp installed but `containerized=yes` is set in the host inventory.

For now, I would say to just use chrony by default only if `is_atomic == True`